### ISSUE TITLE
DeprecationWarning: invalid escape sequence '\`'

### ISSUE THREAD

**Issue by adamchainz** [box/box-python-sdk-gen#329]
- [x] I have checked that the [SDK documentation][sdk-docs] doesn't solve my issue.
- [x] I have checked that the [API documentation][api-docs] doesn't solve my issue.
- [x] I have searched the [Box Developer Forums][dev-forums] and my issue isn't already reported (or if it has been reported, I have attached a link to it, for reference).
- [x] I have searched [Issues in this repo][github-repo] and my issue isn't already reported.

### Description of the Issue

After upgrading my app from boxsdk to box-sdk-gen, I am seeing this warning:

> `DeprecationWarning: invalid escape sequence '\`'`

It happens in these locations in my test run:

```
box_sdk_gen/managers/files.py:531
box_sdk_gen/managers/folders.py:819
box_sdk_gen/managers/folders.py:884
```

I blogged about this warning here: https://adamj.eu/tech/2022/11/04/why-does-python-deprecationwarning-invalid-escape-sequence/ . It becomes an always-shown `SyntaxWarning` on Python 3.12.

The locations all have the same docstring fragment describing a file name parameter:

```
        There are some restrictions to the file name. Names containing
        non-printable ASCII characters, forward and backward slashes
        (`/`, `\`), as well as names with trailing spaces are
        prohibited.
```

The issue is the backslash there, which signifies an escape sequence, but the backtick is not a valid one.

Two possible fixes:

1. Escape the backslash by doubling it.
2. Use raw docstrings by writing `r"""` at the start.

### Steps to Reproduce

Run any of the linked functions with warnings enabled.

### Expected Behavior

No warning.

### Screenshots

n/a

### Versions Used

Python SDK: 1.5.1
Python: 3.11

[sdk-docs]: https://github.com/box/box-python-sdk-gen/tree/main/docs
[api-docs]: https://developer.box.com/docs
[dev-forums]: https://community.box.com/t5/Platform-and-Development-Forum/bd-p/DeveloperForum
[github-repo]: https://github.com/box/box-python-sdk-gen/search?type=Issues

**Comment by adamchainz** [comment]
I started taking a look at fixing this but the box-codegen tool [linked in the contributing guide](https://github.com/box/box-python-sdk-gen/blob/main/CONTRIBUTING.md#how-to-contribute) is not public.

**Comment by lukaszsocha2** [comment]
Hi @adamchainz,
thanks for reporting this. No worries, we will take care of this. Unfortunately box-codegen is still a private repo, but it is planned to make it public in the future.
Best,
@lukaszsocha2

**Comment by mwwoda** [comment]
should be fixed by #330

**Comment by mwwoda** [comment]
Fix for this will be released in the next version of the SDK
